gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dnumslider_ttt2.lua

```diff
@@ -313,7 +313,8 @@ function PANEL:SetServerConVar(cvar)
 	self.serverConVar = cvar
 
 	cvars.ServerConVarGetValue(cvar, function (wasSuccess, value, default)
-		if wasSuccess then
+		-- Check if self is valid before calling SetValue.
+		if wasSuccess and IsValid(self) then
```

Comment:
Same check could be done in dcombobox_ttt2.lua and dtextentry for cvars.ServerConVarGetValue.

Reply:
all right